test/monitoring_console/monitoring_console_test.go

```diff
@@ -508,40 +502,228 @@ var _ = Describe("Monitoring Console test", func() {
+
+			// Deploy Monitoring Console Pod
+			mcTwo, err := deployment.DeployMonitoringConsole(mcTwoName, "")
+			Expect(err).To(Succeed(), "Unable to deploy Monitoring Console instance")
+
+			// Verify Monitoring Console TWO is Ready and stays in ready state
+			testenv.VerifyMonitoringConsoleReady(deployment, mcTwoName, mcTwo, testenvInstance)
+
+			// ###########   VERIFY MONITORING CONSOLE TWO AFTER CLUSTER MASTER RECONFIG  ###################################
+
+			// Check Cluster Master in Monitoring Console Two Config Map
+			testenvInstance.Log.Info("Verify Cluster Master in Monitoring Console Two Config Map after Cluster Master Reconfig")
+			testenv.VerifyPodsInMCConfigMap(deployment, testenvInstance, []string{fmt.Sprintf(testenv.ClusterMasterServiceName, deployment.GetName())}, "SPLUNK_CLUSTER_MASTER_URL", mcTwoName, true)
+
+			// Check Monitoring console Two is configured with all Indexer in Name Space
+			testenvInstance.Log.Info("Verify Indexers in Monitoring Console Pod TWO Config String after Cluster Master Reconfig")
+			testenv.VerifyPodsInMCConfigString(deployment, testenvInstance, indexerPods, mcTwoName, true, true)
+
+			// Check Deployer NOT in Monitoring Console TWO Config Map
+			testenvInstance.Log.Info("Verify DEPLOYER NOT on Monitoring Console TWO Config Map after Cluster Master Reconfig")
+			testenv.VerifyPodsInMCConfigMap(deployment, testenvInstance, []string{fmt.Sprintf(testenv.DeployerServiceName, deployment.GetName())}, "SPLUNK_DEPLOYER_URL", mcTwoName, false)
+
+			// Check Monitoring Console TWO is NOT configured with Search Head in namespace
+			testenvInstance.Log.Info("Verify Search Head Pods NOT on Monitoring Console TWO Config Map after Cluster Master Reconfig")
+			testenv.VerifyPodsInMCConfigMap(deployment, testenvInstance, shPods, "SPLUNK_SEARCH_HEAD_URL", mcTwoName, false)
+
+			testenvInstance.Log.Info("Verify Search Head Pods NOT on Monitoring Console TWO Pod after Cluster Master Reconfig")
+			testenv.VerifyPodsInMCConfigString(deployment, testenvInstance, shPods, mcTwoName, false, false)
+
+			// ##############  VERIFY MONITORING CONSOLE ONE AFTER CLUSTER MASTER RECONFIG #######################
+
+			// Verify Monitoring Console One Ready and stays in ready state
+			testenv.VerifyMonitoringConsoleReady(deployment, deployment.GetName(), mc, testenvInstance)
+
+			// Check Cluster Master Not in Monitoring Console One Config Map
+			testenvInstance.Log.Info("Verify Cluster Master NOT in Monitoring Console One Config Map after Cluster Master Reconfig")
+			testenv.VerifyPodsInMCConfigMap(deployment, testenvInstance, []string{fmt.Sprintf(testenv.ClusterMasterServiceName, deployment.GetName())}, "SPLUNK_CLUSTER_MASTER_URL", mcName, false)
+
+			// Check Monitoring console One is NOT configured with all Indexer in Name Space
+			testenvInstance.Log.Info("Verify Indexers NOT in Monitoring Console One Pod Config String after Cluster Master Reconfig")
```

Comment:
So we will disable this check till CSPL-619 is fixed?

Reply:
Yes